### PR TITLE
Update APM/DBM docs with more features

### PR DESCRIPTION
**Pull request by iFallUpHill**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 1]
```diff
@@ -353,26 +353,35 @@ Enable the database monitoring propagation feature by setting the following envi
...
-## Explore the APM Connection
+## Explore the APM Connection in DBM
 
 ### Attribute active database connections to the calling APM services
 
 {{< img src="database_monitoring/dbm_apm_active_connections_breakdown.png" alt="View active connections to a database broken down by the APM Service they originate from.">}}
 
 Break down active connections for a given host by the upstream APM services making the requests. You can attribute load on a database to individual services to understand which services are most active on the database. Pivot to the most active upstream service’s service page to continue the investigation.
 
+### Filter the Databases List for specific APM calling services.
```
Comment (EmmaOnThursday) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:364]: can this get a short blurb that just says like "filter your database hosts according to what services call them" but... nicer?

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 2]
```diff
@@ -353,26 +353,35 @@ Enable the database monitoring propagation feature by setting the following envi
...
-### Identify the downstream database hosts of APM services
+## Explore the DBM Connection in APM
+
+### Visualize the downstream database hosts of APM services
 
 {{< img src="database_monitoring/dbm_apm_service_page_db_host_list.png" alt="Visualize the downstream database hosts that your APM Services depend on from the Service Page.">}}
 
 On the APM Service Page, view the direct downstream database dependencies of the service as identified by Database Monitoring. Quickly determine if any hosts have disproportionate load that may be caused by noisy neighbors.
 
+### Identify inefficiencies using explain plans for database queries in traces
```
Comment (EmmaOnThursday) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:382]: I feel like there's a lot more going on in this image/tab than the explain plans. Maybe also needs an explainer section that talks about being able to view query info and database health info within trace panel?

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md (hunk 1 above)
Comment (iFallUpHill) [reply content/en/database_monitoring/guide/connect_dbm_and_apm.md]: Updated to friendly wording

**Review by EmmaOnThursday — APPROVED** [review]
love it

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 3]
```diff
@@ -353,26 +353,39 @@ Enable the database monitoring propagation feature by setting the following envi
...
 
 {{< img src="database_monitoring/dbm_apm_active_connections_breakdown.png" alt="View active connections to a database broken down by the APM Service they originate from.">}}
 
 Break down active connections for a given host by the upstream APM services making the requests. You can attribute load on a database to individual services to understand which services are most active on the database. Pivot to the most active upstream service’s service page to continue the investigation.
 
+### Filter your database hosts by the APM services that call them
+
+{{< img src="database_monitoring/dbm_filter_by_calling_service.png" alt="Filter your database hosts by the APM services that call them.">}}
+
+Quickly filter the Database List to display only the database hosts that your specific APM services depend on. Easily identify if any of your downstream  dependencies have blocking activity that may be impacting service performance.
```
Comment (kayayarai) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:368]:
```suggestion
Quickly filter the Database List to display only the database hosts that your specific APM services depend on. Easily identify if any of your downstream dependencies have blocking activity that may impact service performance.   
```

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 4]
```diff
@@ -353,26 +353,39 @@ Enable the database monitoring propagation feature by setting the following envi
...
 
 {{< img src="database_monitoring/dbm_apm_service_page_db_host_list.png" alt="Visualize the downstream database hosts that your APM Services depend on from the Service Page.">}}
 
 On the APM Service Page, view the direct downstream database dependencies of the service as identified by Database Monitoring. Quickly determine if any hosts have disproportionate load that may be caused by noisy neighbors.
 
+### Identify potential optimizations using explain plans for database queries in traces 
+
+{{< img src="database_monitoring/dbm_explain_plans_in_traces.png" alt="Identify inefficiencies using explain plans for database queries within traces.">}}
+
+View historical performance of similar queries to those executed in your trace, including sampled wait events, average latency, and recently captured explain plans, to contextualize how a query is expected to perform. Determine if the behavior is abnormal and continue the investigation by pivoting to Database Monitoring for additional context about the underlying database host(s).
```
Comment (kayayarai) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:388]:
```suggestion
View historical performance of similar queries to those executed in your trace, including sampled wait events, average latency, and recently captured explain plans, to contextualize how a query is expected to perform. Determine if the behavior is abnormal and continue the investigation by pivoting to Database Monitoring for additional context about the underlying database hosts.
```

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 5]
```diff
@@ -353,26 +353,39 @@ Enable the database monitoring propagation feature by setting the following envi
...
+
+### Visualize the downstream database hosts of APM services
 
 {{< img src="database_monitoring/dbm_apm_service_page_db_host_list.png" alt="Visualize the downstream database hosts that your APM Services depend on from the Service Page.">}}
 
 On the APM Service Page, view the direct downstream database dependencies of the service as identified by Database Monitoring. Quickly determine if any hosts have disproportionate load that may be caused by noisy neighbors.
 
+### Identify potential optimizations using explain plans for database queries in traces 
+
+{{< img src="database_monitoring/dbm_explain_plans_in_traces.png" alt="Identify inefficiencies using explain plans for database queries within traces.">}}
```
Comment (kayayarai) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:386]:
```suggestion
{{< img src="database_monitoring/explain_plans_in_traces.png" alt="Identify inefficiencies using explain plans for database queries within traces.">}}
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**